augur/analyze.py

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
```suggestion
"""Data analysis module"""
```

The standard pythonic thing to do here is to use a multiline string. Also, if you don't have more to say than the comment above, i'd just leave it out. Usually people put a description of the module, its purpose and some of the key APIs here.